### DISCUSSION TITLE
How to use the new features? [Part key]

### DISCUSSION THREAD

**Discussion by opendatamarket** [milvus-io/milvus#25735] [Q&A and General discussion]
If part key is used, part name cannot be defined. If querying, how should it be used? Do you have any specific Java documentation to explain?

before querying partitions, loadPart()/search()/releasePart() was required. Now, how to use it specifically？

**Comment by xiaofan-luan** [comment]
> If part key is used, part name cannot be defined. If querying, how should it be used? Do you have any specific Java documentation to explain?
> 
> before querying partitions, loadPart()/search()/releasePart() was required. Now, how to use it specifically？

you can simply specify expression when query, like partitionkey = "aaa"

**Comment by yhmo** [comment]
An explanation is in this thread: https://github.com/milvus-io/milvus/discussions/25650
